src/test/java/apoc/load/relative/LoadCsvRelativePathTest.java

```diff
@@ -0,0 +1,172 @@
+package apoc.load.relative;
+
+import apoc.load.LoadCsv;
+import apoc.util.TestUtil;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mortbay.util.StringMap;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Result;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static apoc.util.MapUtil.map;
+import static apoc.util.TestUtil.testResult;
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class LoadCsvRelativePathTest {
```

Comment:
to just test the correct loading we don't need a full copy of each test file.

a single simple test if the file is found and readable (e.g. by reading all rows is good enough)
same goes for JSON/XML